src/main/resources/locales/en/case-name-search-results.json

```diff
@@ -3,6 +3,7 @@
   "resultsCount": " result(s) successfully found",
   "backButton": "Back",
   "table": {
+    "column0Header": "Select a Result",
```

Comment:
I believe result should be a lower case R

Reply:
Have changed it back again - why do mummy and daddy keep fighting :(